README.md

```diff
@@ -15,3 +15,8 @@ Issue content goes here
 
 /cc @twolfson
 ```
+
+## Attribution
+We use the following icons for our organization and repository logos:
+- [Gear icon](https://thenounproject.com/search/?q=gear&i=102547) by Jardson Almeida, from the Noun Project
+- [Headphones](https://thenounproject.com/search/?q=headphones&i=16961) icon Molly Bramlet, from the Noun Project
```

Comment:
I think the gist is fine. I was mostly complaining about us not using `[* icon]` consistently and missing a "by"